apps/simple-trading-app/src/app/components/simple-market-list/simple-market-toolbar.spec.tsx

```diff
@@ -31,13 +30,18 @@ describe('SimpleMarketToolbar', () => {
   });
 
   it('should be properly rendered', async () => {
-    await act(async () => {
-      render(
-        <MockedProvider mocks={[filterMock]} addTypename={false}>
-          <SimpleMarketToolbar />
-        </MockedProvider>
+    render(
+      <MockedProvider mocks={[filterMock]} addTypename={false}>
+        <SimpleMarketToolbar />
+      </MockedProvider>
+    );
+
+    await waitFor(() => {
+      expect(screen.getByTestId('market-products-menu').children).toHaveLength(
+        3
       );
-      await new Promise((resolve) => setTimeout(resolve, 0));
+      expect(screen.getByTestId('market-assets-menu').children).toHaveLength(6);
+      expect(screen.getByRole('combobox').children).toHaveLength(10);
```

Comment:
Chore (non-blocking): These length values you're expecting are a bit opaque, perhaps they could be documented through var naming or commenting

Reply:
I'm actually not sure what is happening in these tests, I didn't write them, what happened here was I noticed they were failing, and I fixed them by moving things around and removing some hacks which were not working.

Reply:
ended up removing the test changes here, because they got fixed on master meanwhile.